modules/deploy/pages/deployment-option/cloud/managed-connectors/transforms/smt-replacetimestamp.adoc

```diff
@@ -0,0 +1,41 @@
+= ReplaceTimestamp (Redpanda)
+:description: How to set ReplaceTimestamp (Redpanda) Single Message Transform for a connector.
+:page-cloud: true
+
+The ReplaceTimestamp (Redpanda) SMT is designed to support using a record key/value field as a record timestamp, which then can be used to partition data with S3 connector.
+
+Use the concrete transformation type designed for the record key (`com.redpanda.connectors.transforms.ReplaceTimestamp$Key`) or value (`com.redpanda.connectors.transforms.ReplaceTimestamp$Value`).
+
+NOTE: ReplaceTimestamp is available for Sink connector only.
+
+== Configuration
+
+|=== 
+| Property key | Description
+
+| `field`
+| Specifies the name of a field to be used as a source of timestamp.
+|===
+
+== Example
+
+To use `my-timestamp` field as a source of the timestamp for the record, update a connector config with:
+
+----
+"transforms": "ReplaceTimestamp",
+"transforms.ReplaceTimestamp.type": "com.redpanda.connectors.transforms.ReplaceTimestamp$Value",
+"transforms.ReplaceTimestamp.field": "my-timestamp"
+----
+
+for messages in a format:
+
+----
+{
+"name": "my-name",
+...
+"my-timestamp": 1707928150868,
+...
+}
+----
+
+The SMT needs structured data to be able to extract the field from it, which means either a JSON without schema, or JSON/Avro with schema. The timestamp value should be of a numeric type (epoch millis), or a Java Date object (which is the case when using `"connect.name":"org.apache.kafka.connect.data.Timestamp"` in schema).
```

Comment:
We can stick to the terminology from others SMTs, like ExtractField and use `Struct when schema present` instead of `JSON/Avro with schema` and `Map in the case of schemaless data` instead of `JSON without schema`, for consistency.